pkg/utils/util.go

```diff
@@ -24,12 +24,17 @@ import (
 
 	"github.com/openshift/backplane-cli/internal/github"
 	"github.com/openshift/backplane-cli/pkg/info"
+
+	"github.com/andygrunwald/go-jira"
+	"github.com/spf13/viper"
 )
 
 const (
 	ClustersPageSize             = 50
 	BackplaneAPIURLRegexp string = `(?mi)^https:\/\/api\.(.*)backplane\.(.*)`
 	ClusterIDRegexp       string = "/?backplane/cluster/([a-zA-Z0-9]+)/?"
+	JiraTokenConfigKey           = "jira_token"
+	JiraBaseURL                  = "https://issues.redhat.com"
```

Comment:
IMO, We shouldn't hardcode the internal JIRA in public git repo 
You can get the base URL via jira client with a valid token

Reply:
The `JiraTokenConfigKey` is used with `viper` to retrieve the token from the config. So this one will remain a constant there.

For `JiraBaseURL`, this is used in case the CLI user calls option `--notification-issue` with a URL (ex: `https://issues.redhat.com/browse/SDAINT-9165`) instead of a JIRA issue ID (`SDAINT-9165`). I will see if maybe some proper regexp matching could be used to get the issue ID; if not: yes I could indeed use the JIRA client to get that URL prefix info; thanks for the tip.

Reply:
Also, concerning `JiraBaseURL`, the constant is used to initialise the JIRA client... so we definitely cannot get rid of it I guess.

Reply:
You are right, @Nikokolas3270 , We may need to specify `JiraBaseURL` 
As a hack, we can move the Jira base URL to the backplane config as well. 
You can decide what's best in this case :)

Reply:
Agreed, put in the config